source/hierarchy.txt

```diff
@@ -201,15 +208,20 @@ by the same team in the same project, as shown in the following diagram:
    :align: center
    :lightbox:
 
-Grouping multiple {+clusters+} and therefore applications into one
-project by environment as shown in the following diagram simplifies
-administration when the same team is responsible for multiple
-applications across environments. This eases the setup cost for
-features such as private endpoints and customer-managed keys, since 
-all {+clusters+} in the same project share this configuration. However,
-this {+cluster+} hierarchy may violate the least-privilege rule.
+When to Consider Multiple Clusters Per Project 
+``````````````````````````````````````````````
+
+Grouping multiple {+clusters+} into one project by environment 
+as shown in the following diagram simplifies
+administration when one application uses multiple backing clusters, or 
+the same team is responsible for multiple applications across environments. 
+This eases the setup cost for features such as private endpoints and customer-managed keys, 
+since all {+clusters+} in the same project share this configuration. 
+However, this {+cluster+} hierarchy may violate the priniciple of least privilege, 
+which states that users should only be granted the least level of access necessary for their
+role.
 
-You should use this hierarchy only if both of the following are true:
+You should group multiple {+clusters+} in one project only if both of the following are true:
```

Comment:
Suggest:
Add multiple {+clusters+} in one project to groups (staging, production), only if both of the following conditions are true:

Reply:
Use "Each" not "Every" in line 226

Reply:
Also, not sure we want to use "lower" environments, on line 228. If "development" or "test" environments are implied, then spell this out. Users who will read this are the admins who administer these environments. This is their daily work. 

The sentence on line 229 that starts with Production -- could use an active voice.